app/controllers/posts_controller.rb

```diff
@@ -25,6 +25,22 @@ def create
     end
   end
 
```

Comment:
- [ ] It would be better if we use the `CanCanCan` for post authorization in our Post Controller. We can achieve that by adding this piece of code `load_and_authorize_resource` to our Post Controller

Reply:
Thanks @addod19 for your constructive feedback. I have already implemented the changes.